src/script/src/gesi.ts

```diff
@@ -89,15 +89,29 @@ function setMainCharacter() {
 // @ts-ignore
 function reset() {
   const ui = SpreadsheetApp.getUi();
-  const response = ui.alert('Reset?', 'Are you sure you want to reset your data?', ui.ButtonSet.YES_NO);
 
-  if (response === ui.Button.NO) return;
+  const response = ui.alert('Reset?', 'Are you sure you want to reset ALL character data in this project?', ui.ButtonSet.YES_NO);
 
-  Object.keys(getAuthenticatedCharacters()).forEach((characterName: string) => {
+  if (response !== ui.Button.YES) { ui.alert("Reset aborted"); return; }
```

Comment:
```suggestion
  if (response !== ui.Button.YES) { return; }
```
Given they either click away or `no`, don't think we need to do another alert.